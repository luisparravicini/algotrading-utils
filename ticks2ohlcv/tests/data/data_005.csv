1449999960,126.00000,0.40000000
1449999960,130.00000,0.40000000
1449999961,120.00000,0.40000000
1450000020,120.00000,0.40000000
