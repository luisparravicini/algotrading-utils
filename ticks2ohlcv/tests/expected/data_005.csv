date,open,high,low,close,volume
1449999960,126.0000,130.0000,120.0000,120.0000,0.4000
1450000020,120.0000,120.0000,120.0000,120.0000,0.4000
